docs/spec/callables.rst

```diff
@@ -696,19 +694,19 @@ parameters in ``B``::
         def __call__(self, a: int, b: str) -> None: ...
 
     def func(int_kwargs: IntKwargs, int_str_kwargs: IntStrKwargs, str_kwargs: StrKwargs):
-        f1: KwOnly = int_kwargs  # Error: str is not subtype of int
+        f1: KwOnly = int_kwargs  # Error: str is not assignable to int
         f2: KwOnly = int_str_kwargs  # OK
         f3: KwOnly = str_kwargs  # OK
-        f4: IntStrKwargs = str_kwargs  # Error: int | str is not subtype of str
-        f5: IntStrKwargs = int_kwargs  # Error: int | str is not subtype of int
+        f4: IntStrKwargs = str_kwargs  # Error: int | str is not assignable to str
+        f5: IntStrKwargs = int_kwargs  # Error: int | str is not assignable to int
         f6: StrKwargs = int_str_kwargs  # OK
-        f7: StrKwargs = int_kwargs  # Error: str is not subtype of int
+        f7: StrKwargs = int_kwargs  # Error: str is not assignable to int
         f8: IntKwargs = int_str_kwargs  # OK
-        f9: IntKwargs = str_kwargs  # Error: int is not subtype of str
+        f9: IntKwargs = str_kwargs  # Error: int is not assignable to str
         f10: Standard = int_str_kwargs  # Error: Does not accept positional arguments
         f11: Standard = str_kwargs  # Error: Does not accept positional arguments
 
-Subtyping relationships for callable signatures that contain a ``**kwargs``
+Assignability relationships for callable signatures that contain a ``**kwargs``
```

Comment:
Minor: "Assignability relationships" is odd; perhaps change to "Assignability rules".